how-to/virtualisation/libvirt.md

```diff
@@ -175,6 +175,56 @@ There are various options to those methods, but the entry point for all of them
 
 Some useful documentation on the constraints and considerations of live migration can be found at the [Ubuntu Wiki](https://wiki.ubuntu.com/QemuKVMMigration).
 
+## CPU model and topology
+
+libvirt abstracts CPU configuration and provides several options to specify the VM CPU model in the domain definition:
+
+ - custom
+ - host-model
+ - host-passthrough
+ - maximum
+
+While most of thee modes are straightforward, the behavior of `host-model` is more subtle. There is no direct
+translation of `host-model` into a single QEMU `-cpu` argument. Instead, libvirt selects a baseline CPU model and appends
+a list of features:
+
+```bash
+-cpu Haswell-noTSX-IBRS,vmx=on,pdcm=off,...,vmx-entry-load-efer=on,vmx-eptp-switching=on
+```
+
+Because libvirt may not include every known CPU model, it chooses the model that shares the largest set of features with
```

Comment:
```suggestion
Because libvirt can not include every known CPU model and variant, it chooses the model that shares the largest set of features with
```